app/javascript/controllers/admin_date_range_controller.js

```diff
@@ -1,4 +1,4 @@
-import { Controller } from "stimulus"
+import { Controller } from "@hotwired/stimulus"
```

Comment:
To be able to use outlets, stimulus had to be updated. In the update, the package name for stimulus changed.